airbyte-integrations/connectors/source-wrike/source_wrike/manifest.yaml

```diff
@@ -19,7 +19,7 @@ definitions:
     type: "DefaultPaginator"
     pagination_strategy:
       type: "CursorPagination"
-      cursor_value: "{{ last_records['nextPageToken'] }}"
```

Comment:
do we know if this even worked? I'd expect `last_records` to be an array, not an object